src/app/components/harvest/pages/list/list.component.html

```diff
@@ -42,6 +42,26 @@ <h1>Upload History</h1>
       <a class="btn btn-sm btn-primary" [bawUrl]="row.viewUrl">
         {{ asHarvest(row).status !== "complete" ? "Continue" : "View" }}
       </a>
+      <a
+        *ngIf="asHarvest(row).status !== 'complete'"
```

Comment:
move this check to a function

it also needs to be a more complex check: as per https://github.com/QutEcoacoustics/baw-server/blob/0e54cbd30ae9fc704466cce11fbd1ba4de8e674e/app/models/harvest.rb#L395-L399 updating a harvest is only allowed in certain states.